README.md

```diff
@@ -285,6 +285,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [AvidCoder101](https://github.com/AvidCoder101)
   - [Avinash Prasad](https://github.com/avionmission)
   - [Avneesh Agarwal](https://github.com/avneesh0612)
+  -[AyoOluwa Israel Adeleke](https://github.com/AyoOluwa-Israel)
```

Comment:
```suggestion
  - [AyoOluwa Israel Adeleke](https://github.com/AyoOluwa-Israel)
```

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix